test/common/README.md

```diff
@@ -53,23 +53,21 @@ Check if there is more than 1gb of total memory.
 ### expectsError([fn, ]settings)
 * `fn` [&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function)
 * `settings` [&lt;Object>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
-  with the following optional properties:
-  * `code` [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
-    expected error must have this value for its `code` property
+  that may contain any optional property. Each property will be checked by
+  strict equality besides:
   * `type` [&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function)
-    expected error must be an instance of `type`
+    expected error must be an instance of `type` and should be [subclassed by
+    the `internal/errors` module](https://github.com/nodejs/node/blob/master/doc/guides/using-internal-errors.md#api).
```

Comment:
This additional bit `subclassed by the 'internal/errors' module` doesn't really make sense. The thing must be an instance of `type` and must be an `Error` subclass. Whether it is a subclass of one of the `internal/errors.js` classes is somewhat inconsequential.